.opspec/release/to-ghcr/check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+apk add --update docker
+
+# only continue with release if tag doesn't exist
+if docker pull ghcr.io/opctl/opctl:${version}-dind; then
```

Comment:
nit) It could be nice to check for release without taking a dep on docker since we don't specifically rely on docker (curling GitHub releases API for example) but not a blocker.

Reply:
yeah only a nit; like you said we could revisit later if we want